interactive_engine/compiler/src/test/java/com/alibaba/graphscope/common/ir/planner/rbo/ScanEarlyStopTest.java

```diff
@@ -0,0 +1,110 @@
+/*
+ *
+ *  * Copyright 2020 Alibaba Group Holding Limited.
```

Comment:
Remove the first * of The license section.

Reply:
fixed in https://github.com/alibaba/GraphScope/pull/4431/commits/c55ec36f31445bb51775fb4c687105438b6aa9ea